packages/app/src/app/pages/Dashboard/Content/routes/PathedSandboxes/index.js

```diff
@@ -31,8 +33,28 @@ const PathedSandboxes = props => {
                 ? []
                 : data.me.collection.sandboxes;
 
-            const possibleTemplates = uniq(
-              sandboxes.map(x => x.source.template)
+            const possibleTemplates = uniqBy(
+              sandboxes.map(x => {
```

Comment:
@CompuIves I think this would be a bit cleaner if you destructur `x` here and not having `x` all over this map function.

```js
sandboxes.map(({ forkedTemplate, source }) => {
```